### PR TITLE
OneStar Guns and Uniforms Part 2 [READY]

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request

This PR adds several onestar items to the game, the Type 21 Yaogui shotgun, the OneStar Industrial voidsuit and rebalances the other two onestar guns. they are currently adminspawn only. the shotgun was approved by firefox and shazbot.
29-Sep-22 15-58-53

## Why It's Good For The Game
more onestar content to use in the future is good

## Changelog
:cl:
add: Type 21 Shotgun and Onestar Voidsuit
balance: rebalanced the onestar rifles
imageadd: Type 21 and OS Voidsuit Sprites
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
why did it fail the cheeeeeeecks

**Comment by ValoTheValo** [comment]
PR Is ready

**Comment by Pink-Chink** [comment]
image

You should move this to a clean branch.

**Comment by ValoTheValo** [comment]
> image
> 
> You should move this to a clean branch.

i dont even know how to do that or why thats even happening

File: code/modules/projectiles/guns/projectile/shotgun/type21.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	magazine_type = /obj/item/ammo_magazine/m12
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 10)
+	price_tag = 2000
+	spawn_blacklisted = TRUE //until onestar rework
+	fire_sound = 'sound/weapons/guns/fire/shotgunp_fire.ogg'
+	unload_sound = 'sound/weapons/guns/interact/ltrifle_magout.ogg'
+	reload_sound = 'sound/weapons/guns/interact/ltrifle_magin.ogg'
+	cocked_sound = 'sound/weapons/guns/interact/hpistol_cock.ogg'
+	damage_multiplier = 1.0
+	penetration_multiplier = 0.5 // this is not babies first gun. It's a Serb-level weapon.
```
Comment (DiaFRAME444) [inline code/modules/projectiles/guns/projectile/shotgun/type21.dm:23]: Sir this "One Star Shotgun" is just the Bojevic.

Comment (ValoTheValo) [reply code/modules/projectiles/guns/projectile/shotgun/type21.dm:23]: stats are somewhat changed and its literally the only other magfed semi auto shotgun in the game

Comment (ValoTheValo) [reply code/modules/projectiles/guns/projectile/shotgun/type21.dm]: there removed the comment

**Comment by Pink-Chink** [comment]
I would say rebase, but I think it might bring the changelog edits with it. Sync your master, save your edited files, make a new branch, and put your stuff in the new branch.

**Comment by SirRichardFrancis** [comment]
> i dont even know how to do that or why thats even happening

Daily reminder to use normal software and check out new branches, while keeping master branch clean.
If that is not something you are willing to do, at least add changelogs to .gitignore.
Wouldn't solve problems like morbillion unrelated commits in a PR, but it's something.

**Comment by ValoTheValo** [comment]
> > i dont even know how to do that or why thats even happening
> 
> Daily reminder to use normal software and check out new branches, while keeping master branch clean. If that is not something you are willing to do, at least add changelogs to .gitignore. Wouldn't solve problems like morbillion unrelated commits in a PR, but it's something.

there is only like 1 changelog thing. and I'm using VSC and Github desktop.

**Comment by ValoTheValo** [comment]
how do i even get rid of the changelog stuff anyway

**Comment by SirRichardFrancis** [comment]
> there is only like 1 changelog thing. and I'm using VSC and Github desktop.
> Github desktop
> **_Github desktop_**

It's been know to work poorly with Eris repo for years now.
And you personally bit that cactus multiple times before. Why still use it?
https://youtu.be/BCfoK6RFW8c

**Comment by ValoTheValo** [comment]
> > there is only like 1 changelog thing. and I'm using VSC and Github desktop.
> > Github desktop
> > **_Github desktop_**
> 
> It's been know to work poorly with Eris repo for years now. And you personally bit that cactus multiple times before. Why still use it? https://youtu.be/BCfoK6RFW8c

gitkraken is worse and all the other programs are worse and require waaaaay to much fuckery than im willing to deal with

**Comment by ValoTheValo** [comment]
i think that fixed it

**Comment by SirRichardFrancis** [comment]
> gitkraken is worse and all the other programs are worse and require waaaaay to much fuckery than im willing to deal with

SmartGit.

File: code/modules/clothing/spacesuits/void/offship.dm [hunk 2]
```diff
@@ -0,0 +1,39 @@
+//voidsuits found in exploration. mainly onestar
+/obj/item/clothing/head/space/void/onestar
+	name = "Onestar Voidsuit Helmet"
+	desc = "a lightweight bubble helmet providing unrestricted vision and protection from space with a light on top. the glass is plasma reinforced and could easily stop a rifle round"
```
Comment (Pink-Chink) [inline code/modules/clothing/spacesuits/void/offship.dm:4]:
```suggestion
	desc = "A lightweight bubble helmet providing unrestricted vision and protection from space with a light on top. The glass is plasma-reinforced and could easily stop a rifle round."
```

File: code/modules/projectiles/guns/projectile/shotgun/type21.dm [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/obj/item/gun/projectile/shotgun/type_21
+	name = "OS Type 21 SG \"Yaoguai\"" //Demon
+	desc = "an early model shotgun used by onestar. its compact with a sawn down barrel and no stock. and designed for police use in megacities its recoil kicks badly but the stopping power is worth it. its fed from drum magazines"
```
Comment (Pink-Chink) [inline code/modules/projectiles/guns/projectile/shotgun/type21.dm:3]: Grammar is a bit funky here.

**Comment by Pink-Chink** [comment]
For the names and descriptions, it should be "One Star", not "Onestar".

**Comment by SirRichardFrancis** [comment]
Or "Alliance", aka "Alliance of Humanity", which is essentially the same thing as "One Star" iirc.

File: code/modules/clothing/spacesuits/void/offship.dm (hunk 2 above)
Comment (ValoTheValo) [reply code/modules/clothing/spacesuits/void/offship.dm]: commited

**Comment by ValoTheValo** [comment]
> Or "Alliance", aka "Alliance of Humanity", which is essentially the same thing as "One Star" iirc.

i have no idea if Alliance of Humanity is a depreciated term in lore or not

File: code/modules/clothing/spacesuits/void/offship.dm [hunk 4]
```diff
@@ -0,0 +1,39 @@
+//voidsuits found in exploration. mainly onestar
+/obj/item/clothing/head/space/void/onestar
+	name = "OneStar Voidsuit Helmet"
```
Comment (Pink-Chink) [inline code/modules/clothing/spacesuits/void/offship.dm:3]:
```suggestion
	name = "One Star Voidsuit Helmet"
```

File: code/modules/clothing/spacesuits/void/offship.dm [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+		bomb = 75,
+		bio = 100,
+		rad = 100
+	)
+	siemens_coefficient = 0.35
+	light_overlay = "helmet_light_dual"
+	obscuration = LIGHT_OBSCURATION
+
+/obj/item/clothing/suit/space/void/onestar
+	name = "Onestar Voidsuit"
```
Comment (Pink-Chink) [inline code/modules/clothing/spacesuits/void/offship.dm:20]:
```suggestion
	name = "One Star Voidsuit"
```

File: code/modules/clothing/spacesuits/void/offship.dm [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+	desc = "A OneStar industrial voidsuit designed to be resistant to bluespace anomalies and all types of damage. hundreds of thousands of theese were produced in the dying days of OneStar. but they failed to be enough to save its citizens. the sight of a skeleton lying dead in this voidsuit is not an uncommon sight in nullspace, often horifically warped and damaged. the suit is incredibly light and appears to be made out of a special fibre and lightweight matriels to allow movement of limbs in EVA while not sacrificing protection"
```
Comment (Pink-Chink) [inline code/modules/clothing/spacesuits/void/offship.dm:21]:
```suggestion
	desc = "A One Star industrial voidsuit designed to be resistant to bluespace anomalies and all types of damage. Hundreds of thousands of these were produced in the dying days of One Star, but they failed to be enough to save its citizens. The sight of a skeleton lying dead in this voidsuit is not an uncommon sight in nullspace, often horrifically warped and damaged. The suit is incredibly light and appears to be made out of a special fibre and lightweight materials to allow movement of limbs in EVA while not sacrificing protection."
```

File: code/modules/projectiles/guns/projectile/automatic/type17.dm [hunk 7]
```diff
@@ -1,31 +1,31 @@
 /obj/item/gun/projectile/automatic/type_17
-	name = "OS AR .20 \"Type 17\""
-	desc = "An older model Onestar assault rifle. A reliable, if unintuitive, design. Uses .20 Rifle magazines."
+	name = "OS AR .30 \"Type 17\"" //generic AR. i give up on trying to make this unique
+	desc = "An older model OneStar assault rifle. A reliable, if unintuitive, design. Uses .30 Rifle magazines."
```
Comment (Pink-Chink) [inline code/modules/projectiles/guns/projectile/automatic/type17.dm:3]:
```suggestion
	desc = "An older model One Star assault rifle. A reliable, if unintuitive, design. Uses .30 Rifle magazines."
```

File: code/modules/projectiles/guns/projectile/automatic/type47.dm [hunk 8]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/type_47
-	name = "OS CAR .25 CS \"Type 47\""
-	desc = "A compact assault carbine of onestar orgin."
+	name = "OS Type 47 CAR .25 CS \"Zuosui\"" //Ghost
+	desc = "A boxy carbine of OneStar orgin designed for special forces. there is a giant supressor attached to it."
```
Comment (Pink-Chink) [inline code/modules/projectiles/guns/projectile/automatic/type47.dm:3]:
```suggestion
	desc = "A boxy carbine of One Star origin designed for special forces. There is a giant suppressor attached to it."
```

File: code/modules/projectiles/guns/projectile/shotgun/type21.dm [hunk 9]
```diff
@@ -0,0 +1,47 @@
+/obj/item/gun/projectile/shotgun/type_21
+	name = "OS Type 21 SG \"Yaoguai\"" //Demon
+	desc = "an early model shotgun used by OneStar. its compact with a sawn down barrel and no stock. and designed for police use in megacities its recoil kicks badly but the stopping power is worth it. its fed from drum magazines"
```
Comment (Pink-Chink) [inline code/modules/projectiles/guns/projectile/shotgun/type21.dm:3]:
```suggestion
	desc = "An early model shotgun used by One Star. It is compact with a sawn down barrel and no stock, and designed for police use in megacities. Its recoil kicks badly but the stopping power is worth it. Its fed from drum magazines"
```

**Review by Pink-Chink — APPROVED** [review]
Finally, some OS loot.

**Pull request merged**